app/logic/adminNewEvent.py

```diff
@@ -50,7 +50,31 @@ def createNewEvent(newEventData):
 
             facilitatorEntry = Facilitator.create(user = newEventData['eventFacilitator'],
                                                       event = newEvent)
+        return newEvent
     else:
         raise Exception("Invalid Data")
 
-    return newEvent
+def eventEdit(newEventData):
+

```

Comment:
Use the new `'valid'` key that Sandesh added in his last update to confirm the data is ready to go